static/bidder-info/bmtm.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "https://one.elitebidder.com/api/pbs"
 maintainer:
-  email: dev@brightmountainmedia.com
+  email: product@brightmountainmedia.com
```

Comment:
Sent an email to above address for verification.

Reply:
@bretg We are not able to verify the above email address. Please allow "prebid@microsoft.com" email to post messages to the above email address.

Reply:
Yes, working with that team to get them to fix it